src/web/mjs/connectors/DoujinDesu.mjs

```diff
@@ -17,10 +17,6 @@ export default class DoujinDesu extends WordPressMangastream {
         this.querMangaTitleFromURI = 'section.metadata h1.title';
     }
 
-    canHandleURI(uri) {
-        return /doujindesu\.xxx|212\.32\.226\.234/.test(uri.hostname);
```

Comment:
clipboard doesn't work anymore ?

Reply:
?? this function was useful when we have had more than one domain. Now its handled by default code and i guess it still works 🤷